olm-catalog/serverless-operator/project.yaml

```diff
@@ -6,19 +6,19 @@ project:
   # all components in `dependencies.previous` to the same versions as `dependencies` in the same PR.
   # Otherwise, the upgrade tests will not pass, as we have a different SO version with the same bundle contents.
   # Also make sure to update values under `olm.previous` by copying from `olm.replaces` and `olm.skipRange`.
-  version: 1.29.0
+  version: 1.30.0
```

Comment:
We need to follow the comment above!

/hold

Reply:
https://github.com/openshift-knative/serverless-operator/pull/2073/commits/a0d144a07a9b00ca688b0b933e5f0954ec0a1c40

Reply:
For eventing, I'm pretty sure we can use the same format as `dependencies`

Reply:
Gotcha it now, thanks for more hints on this one. I see Serving is on  the format as Eventing components, I hope it works.